L1Trigger/GlobalCaloTrigger/test/testElectronSorter.cpp

```diff
@@ -305,6 +305,7 @@ void convertToGct(EmInputCandVec candidates) {
         } else {
           eta = 5;
         }
+        [[fallthrough]];
```

Comment:
@mrodozov why ```[[fallthrough]]``` instead of ```break``` as in all the other cases? I would naively imagine it was just forgotten, looking at the code.
@rekovic could you please check?